e2e/support/helpers/e2e-metabot-helpers.ts

```diff
@@ -30,7 +30,7 @@ export function openMetabotViaShortcutKey(assertVisibility = true) {
     assertChatVisibility("not.visible");
   }
 
-  cy.realPress(["Meta", "b"]);
+  cy.get("body").type("{ctrl+b}{cmd+b}");
```

Comment:
This is how we already trigger the command palette, and it seems to be working.

It would be nice to use the realPress for everything, though.